test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -752,6 +753,7 @@ public void ErrorFunction_DoesNotBringDownHost()
             Assert.Equal(testData, (string)resultObject["reqBody"]);
             Assert.Equal("string", (string)resultObject["reqRawBodyType"]);
             Assert.Equal(testData, (string)resultObject["reqRawBody"]);
+            Assert.Equal("text/plain", resultObject["reqHeaders"]["content-type"]);
```

Comment:
Do we have other tests passing multi-value headers, headers with quality, etc.? Would be good to make sure those are handled as expected.